app.js

```diff
@@ -0,0 +1,12 @@
+const NewsApi = require("./src/newsApi")
+const NewsModel = require("./src/newsModel")
+const NewsView = require("./src/newsView")
+
+let api = new NewsApi();
+let model = new NewsModel();
+let view = new NewsView()
+
+view.displayNews((data)=>{
+	model.updateNews(data)
+	console.log(model.getNews());
```

Comment:
Think it might be worthwhile removing console logs as the project is now finished. Good for displaying information while building but once finished this might be extra lines of unnecessary code.